packages/server/lib/html/non_proxied_error.html

```diff
@@ -6,7 +6,46 @@
 
     <link href="/__cypress/runner/favicon.ico?v2" rel="icon">
 
-    <link rel="stylesheet" href="/__cypress/runner/cypress_runner.css">
+    <style>
```

Comment:
It looks like these styles might have been updated , thus the updated screenshot. Was that intentional?

Reply:
It was - I deleted a bunch of styles as part of this PR. The page originally contained all of `cypress_runner.css`, which included all the styles for the entire runner and reporter, which are really not necessary for this simple page.

Instead of going over literally thousands of lines of CSS to figure out the exact thing that was messing up the screenshot, I just committed a new one, that looks almost identical, but the text is a tiny bit smaller. I think this is a good compromise - we remove thousands of lines of otherwise-unused CSS, and the only visual difference is a page is almost never seen, which just shows a basic warning.